src/constants/extensionConfiguration.constants.ts

```diff
@@ -1,10 +1,10 @@
 export const DEFAULT_SERVER_URL = "http://localhost:9980";
 export const EXT_PUBLISHER = "undefined_publisher";
 export const DEFAULT_PROJECT_VIEW_REFRESH_INTERVAL = 1000;
-export const DEFAULT_PROJECT_VIEW_SESSION_LOG_REFRESH_INTERVAL = 1000;
 export const DEFAULT_ENVIRONMENT = "default";
 export const DEFAULT_CONNECTION_TEST_INTERVAL = 5000;
 export const INITIAL_RETRY_SCHEDULE_COUNTDOWN = 60;
+export const INITIAL_SESSIONS_RETRY_SCHEDULE_COUNTDOWN = 5;
```

Comment:
consult with @haimzlato

Reply:
Add an option to modify using the vscode config.

Reply:
After discussion with @haimzlato - the user shouldn't have an access to the interval times.